src/query.ts

```diff
@@ -397,6 +397,10 @@ class Query {
    *     If not specified, default values are chosen by Datastore for the
    *     operation. Learn more about strong and eventual consistency
    *     [here](https://cloud.google.com/datastore/docs/articles/balancing-strong-and-eventual-consistency-with-google-cloud-datastore).
+   * @param {object} [options.gaxOptions] Request configuration options, outlined
+   *     here: https://googleapis.github.io/gax-nodejs/global.html#CallOptions.
+   * @param {boolean | IntegerTypeCastOptions} [options.wrapNumbersOptions=false]
```

Comment:
The description should explain "If a boolean, this will wrap values... If an object, you can customize the behavior..."

Reply:
Does below sounds ok?
```
   * @param {boolean | IntegerTypeCastOptions} [options.wrapNumbers=false]
   *     Wrap values of integerValue type in {@link Datastore#Int} object.
   *     If a `boolean`, this will wrap values in {@link Datastore#Int}.
   *     If an `object`, this will return a value returned by 
   *     `wrapNumbers.integerTypeCastFunction`.
   *     Please see {@link  IntegerTypeCastOptions} for options descriptions.

```

Reply:
Pushed the changes 👍